build/dependencies.yaml

```diff
@@ -154,13 +154,13 @@ dependencies:
       match: tag =
 
   - name: "k8s.gcr.io/pause"
-    version: 3.4
+    version: 3.4.2
```

Comment:
Where is pause 3.4.2 coming from? The rest of this PR is .1? Is this supposed to be the new dev version?

Reply:
As I don't update the image, I think I should keep it as 3.4.1. 
Will fix it.